src/vs/editor/common/config/commonEditorConfig.ts

```diff
@@ -822,6 +822,11 @@ const editorConfiguration: IConfigurationNode = {
 			'default': EDITOR_DEFAULTS.contribInfo.lightbulbEnabled,
 			'description': nls.localize('codeActions', "Enables the code action lightbulb in the editor.")
 		},
+		'editor.longLineLength': {
+			'type': 'integer',
+			'default': EDITOR_DEFAULTS.contribInfo.longLineLength,
```

Comment:
is `contribInfo` a right place for this setting?

Reply:
I think it would go better as EDITOR_DEFAULTS.longLineLength. @alexandrudima, can you confirm?

Reply:
Renamed option name to `maxTokenizationLineLength`

Reply:
god job!